pipeline/acquisition.py

```diff
@@ -0,0 +1,375 @@
+            self.Trial.insert1(key, ignore_extra_fields=True)
+            print(f'{trialId} ',end="")
+            # ======== Now add trial descriptors to the TrialInfo part table ====
+            # - good/bad trial_status (nwb['analysis']['good_trials'])
+            key['trial_status'] = 'good' if good_trials.flatten()[idx] == 1 else 'bad'
+            # - trial_type and trial_stim_status (nwb['epochs'][trial]['description']) 
+            key['trial_type'], key['trial_stim_status'] =  re.split(', ',trial_descs[idx])
+            # - trial_response (nwb['analysis']['trial_type_string'])
+            # note, the last type_string value is duplicated info of "stim"/"no stim" above, so ignore it here (hence the [idx,:-1])
+            match_idx = np.where(trial_type_mat[idx,:-1] == 1)
+            trial_response =  trial_type_string.flatten()[match_idx].item(0).decode('UTF-8')
+            if re.search('correct',trial_response.lower()) is not None:
+                trial_response = 'correct'
+            elif re.search('incorrect',trial_response.lower()) is not None:
+                trial_response = 'incorrect'
+            key['trial_response'] = trial_response.lower()
+            # insert
+            self.TrialInfo.insert1(key, ignore_extra_fields=True)
+        print('')
+    
+    
+@schema
+class BehaviorAcquisition(dj.Imported):
+    definition = """
+    -> Session
+    -> reference.BehavioralType
+    ---
+    behavior_time_stamp: longblob
+    behavior_timeseries: longblob        
+    """    
+      
+    
+@schema
+class ExtracellularAcquisition(dj.Imported):
+    definition = """
+    -> ExtracellularInfo
+    -> reference.ExtracellularType
+    ---
+    ec_time_stamp: longblob
+    ec_timeseries: longblob        
+    """      
+    
+    
+@schema
+class IntracellularAcquisition(dj.Imported):
+    definition = """
+    -> IntracellularInfo
+    -> reference.IntracellularType
+    ---
+    ic_time_stamp: longblob
+    ic_timeseries: longblob        
+    """     
+       
+     
+@schema
+class ExperimentalStimulus(dj.Imported):
+    definition = """
+    -> StimulationInfo
+    ---
+    stim_time_stamp: longblob
+    stim_timeseries: longblob        
+    """      
+            
+    
+@schema
+class TrialExtracellular(dj.Computed):
+    definition = """
+    -> ExtracellularAcquisition
+    -> TrialSet.Trial
+    ---
+    segmented_extracellular: longblob
+    """
+    
+    
+@schema
+class TrialIntracellular(dj.Computed):
+    definition = """
+    -> IntracellularAcquisition
+    -> TrialSet.Trial
+    ---
+    segmented_intracellular: longblob
+    """
+    
+    
+@schema   
+class TrialBehavior(dj.Computed):
+    definition = """
+    -> BehaviorAcquisition
+    -> TrialSet.Trial
+    ---
+    segmented_behavior: longblob
+    """
+    
+    
+@schema
+class TrialStimulus(dj.Computed):
+    definition = """
+    -> ExperimentalStimulus
+    -> TrialSet.Trial
+    ---
+    segmented_stim: longblob
+    """
+    
+    
```

Comment:
Do not leave bunch of blank lines at the end - it makes navigating through code esp during PR harder.